src/devices/src/virtio/block.rs

```diff
@@ -707,13 +707,23 @@ mod tests {
 
             let epoll_config = EpollConfig::new(0, epoll_raw_fd, sender);
 
-            let f: File = tempfile().unwrap();
-            f.set_len(0x1000).unwrap();
+            let tmp_f = TempFile::new().unwrap();
+            tmp_f.as_file().set_len(0x1000).unwrap();
 
             // Rate limiting is enabled but with a high operation rate (10 million ops/s).
             let rate_limiter = RateLimiter::new(0, None, 0, 100_000, None, 10).unwrap();
             DummyBlock {
-                block: Block::new(f, is_disk_read_only, epoll_config, Some(rate_limiter)).unwrap(),
+                block: Block::new(
+                    OpenOptions::new()
```

Comment:
Why not `tmp_f.as_file().try_clone().unwrap()` instead of `OpenOptions::new()..`?